CHANGES.rst

```diff
@@ -1,6 +1,9 @@
 0.11.0 (unreleased)
 ===================
 
+- Dropped ``pytest-openfiles`` as dependency. ``astropy`` no longer
```

Comment:
I would not mix astropy into here, it's being dropped as it's scheduled for retirement and the standard tools are providing the functionality. What astropy core does is a bit irrelevant.

Reply:
I removed the mention.